recipes/geogram/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class GeogramConan(ConanFile):
+    name = "geogram"
+    description = "a programming library with geometric algorithms"
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```
Please use the official SPDX short name only: https://spdx.org/licenses/BSD-3-Clause.html